hazelcast-jet-core/src/test/java/com/hazelcast/jet/SimpleTestInClusterSupport.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet;
+
+import com.hazelcast.core.DistributedObject;
+import com.hazelcast.jet.config.JetClientConfig;
+import com.hazelcast.jet.config.JetConfig;
+import com.hazelcast.jet.core.JetTestSupport;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.runner.RunWith;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * Base class for tests that share the cluster for all jobs. The subclass must
+ * call {@link #beforeClassWithClient} method.
+ */
+@RunWith(HazelcastSerialClassRunner.class)
+public class SimpleTestInClusterSupport extends JetTestSupport {
+
+    private static JetTestInstanceFactory factory;
+    private static JetConfig jetConfig;
+    private static JetInstance[] instances;
+    private static JetInstance client;
+
+    protected static void beforeClass(int memberCount, @Nullable JetConfig jetConfig) {
```

Comment:
can we have more descriptive name for the method? like setMemberCount etc

Reply:
I used `initialize` and `initializeWithClient`